### ISSUE TITLE
Hostile Mobs can attack the player while in a belly

### ISSUE THREAD

**Issue by FartMaster69420** [VOREStation/VOREStation#11520]
#### Brief description of the bug
Eating a hostile mob will lead to it attacking while digesting

#### What you expected to happen
To eat the food

#### What actually happened
Pummeled by the food

#### Steps to reproduce
- Eat a hostile mob, I've seen sheep and Virgo Two Jellies do this, presumably other mobs will too
- The mob will begin attacking while within the gut, potentially even putting it's pred into crit

#### Code Revision
Server revision: B:release D:
Commit: 4c0929aaeb6efb7bd30082ec6a8a9f1348095976 
 
#### Anything else you may wish to add:
- A nice change could be to make the mobs resist, causing resist mechanics to come into play more.

**Comment by Very-Soft** [comment]
This is not a bug, intended to discourage people abusing vore mechanics. If someone's beating you up and you go and eat them, they can still do something about it. Mobs can't really do this, and there's been plenty of times where we set up a boss or whatever, and someone's like 'yes, I eat it' and it's just like. Okay! There's nothing we can do about that except be like 'oh no it's clawing its way out you better throw it up or you're gonna gib!'

So! I figured, instead of gibbing people who eat something that's literally trying to kill you, just have it not stop trying to kill you.
So! You get beat up from the inside. Eating something that's fighting you is a bad idea.
